### ISSUE TITLE
'SemanticTransformerWrapper' object has no attribute 'embed_text'

### ISSUE THREAD

**Issue by jinyuli** [lucidrains/audiolm-pytorch#197]
I'm trying to generate sound with text, but got this:
> AttributeError: 'SemanticTransformerWrapper' object has no attribute 'embed_text'

and I didn't find the method `embed_text` in class `SemanticTransformerWrapper` either.

**Comment by lucidrains** [comment]
@jinyuli oh nice, you must have gotten to the final stage? could you try `1.1.4`?

**Comment by jinyuli** [comment]
thanks, it's fixed.

but I got another issue, while generating audio with text(running on google Colab):
* if cuda is used, then got error like "GET was unable to find an engine to execute", after googling, the cause may be CUDA version incompatible, Colab cuda version is 12.0, Pytorch's is 11.8, I'm not sure whether it's a problem.
* if cpu is used, then got error said that all tensors should use the same device, but I've set all transformer to cpu. the error is from the following code:
```
            logits = self.transformer.forward_with_cond_scale(
                ids = sample_semantic_ids,
                text_embeds = text_embeds,
                cond_scale = cond_scale,
                **kwargs
            )
```
